### PR TITLE
Add the user management section of to the Details section of task layout [SCI-4505]

### PR DESCRIPTION
**Pull request by mlorb**
Jira ticket: [SCI-4505](https://biosistemika.atlassian.net/browse/SCI-4505)

### What was done
Add the user management section of to the Details section of task layout.
It was reused the one from canvas view.

### CODE REVIEW

File: app/assets/javascripts/my_modules.js [hunk 1]
```diff
@@ -290,8 +290,76 @@ function initTagsSelector() {
...
+    $('.task-details .manage-users-link').on('ajax:success', function(e, data) {
+      manageUsersModal.modal('show');
+      manageUsersModal.find('#manage-module-users-modal-module').text(data.my_module.name);
+      initUsersModalBody(data);
+    });
+  }
+
+  // Initialize reloading manage user modal content after posting new user
+  function initAddUserForm() {
+    manageUsersModalBody.find('.add-user-form')
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules.js:308]: Make here delegated event handling

File: app/assets/javascripts/my_modules.js [hunk 2]
```diff
@@ -290,8 +290,76 @@ function initTagsSelector() {
...
+      .on('ajax:success', function(e, data) {
+        initUsersModalBody(data);
+      });
+  }
+
+  // Initialize remove user from my_module links
+  function initRemoveUserLinks() {
+    manageUsersModalBody.find('.remove-user-link')
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules.js:316]: Make here delegated event handling

File: app/assets/javascripts/my_modules.js [hunk 3]
```diff
@@ -290,8 +290,76 @@ function initTagsSelector() {
...
+  // Initialize ajax listeners and elements style on modal body.
+  // This function must be called when modal body is changed.
+  function initUsersModalBody(data) {
+    manageUsersModalBody.html(data.html);
+    manageUsersModalBody.find('.selectpicker').selectpicker();
+    initAddUserForm();
+    initRemoveUserLinks();
+  }
+
+  manageUsersModal = $('#manage-module-users-modal');
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules.js:331]: Maybe init this at top

File: app/assets/javascripts/my_modules.js [hunk 4]
```diff
@@ -290,8 +290,76 @@ function initTagsSelector() {
   }).getContainer(myModuleTagsSelector).addClass('my-module-tags-container');
 }
 
+function initAssignedUsersSelector() {
+  var manageUsersModal = null;
+  var manageUsersModalBody = null;
+
+  // Initialize users editing modal remote loading
+  function initUsersEditLink() {
+    $('.task-details .manage-users-link').on('ajax:success', function(e, data) {
```
Comment (aignatov-bio) [inline app/assets/javascripts/my_modules.js:299]: Make here delegated event handling

File: app/controllers/user_my_modules_controller.rb [hunk 5]
```diff
@@ -19,6 +19,18 @@ def index
     end
   end
 
+  def index_compact
```
Comment (aignatov-bio) [inline app/controllers/user_my_modules_controller.rb:22]: Can you use more descriptive name

File: app/controllers/user_my_modules_controller.rb [hunk 6]
```diff
@@ -19,6 +19,18 @@ def index
...
+    respond_to do |format|
+      format.json do
+        render json: {
+          html: render_to_string(
+            partial: 'index_compact.html.erb'
```
Comment (aignatov-bio) [inline app/controllers/user_my_modules_controller.rb:27]: Same here

File: app/views/user_my_modules/_index_compact.html.erb [hunk 7]
```diff
@@ -0,0 +1,27 @@
+<% user_my_modules = @my_module.user_my_modules %>
+<% if can_manage_users_in_module?(@my_module) %>
+  <a class="task-assigned-users manage-users-link"
+     data-remote="true"
+     href="<%= my_module_users_edit_path(@my_module, format: :json) %>"
+     data-module-id="<%= @my_module.id %>"
+     data-module-users-url="<%= index_compact_my_module_user_my_modules_url(@my_module, format: :json) %>">
+    <% user_my_modules.each_with_index do |user_my_module, i| %>
```
Comment (aignatov-bio) [inline app/views/user_my_modules/_index_compact.html.erb:8]: Why you use index here?

File: app/views/user_my_modules/_index_compact.html.erb [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+        <%= image_tag avatar_path(user, :icon_small) %>
+      </span>
+    <% end %>
+    <span class='global-avatar-container assign-new-user'>
+      <i class="fas fa-plus"></i>
+    </span>
+  </a>
+<% else %>
+  <div class="task-assigned-users">
+    <% user_my_modules.each_with_index do |user_my_module, i| %>
```
Comment (aignatov-bio) [inline app/views/user_my_modules/_index_compact.html.erb:20]: Why you use index here?

File: app/controllers/user_my_modules_controller.rb (hunk 5 above)
Comment (okriuchykhin) [inline app/controllers/user_my_modules_controller.rb:22]: Where is permission check for this action??

**Pull request merged**